virttest/utils_net.py

```diff
@@ -4259,6 +4261,12 @@ def create_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     :param ignore_status: Whether to raise an exception when command fails
     :return: The command status and output
     """
+    runner = process.run
+    if session:
+        runner = RemoteRunner(session=session).run
```

Comment:
Just wondering, is there a way to achieve the same without using the a remote runner? This class was planned to be deprecated and I am not sure what is it that it provides here that a normal remote session won't provide.

Reply:
No, I think RemoteRunner.run() is the only way to manage services with session via create_service() for now. It requires a process.CmdResult instance, so if you want to remove virttest.remote.RemoteRunner, maybe need to update service.FactoryHelper() or add some new one.

BTW, may I ask why you want to deprecate RemoteRunner? We have 11 files in tp-libvirt which call that, some cases will be impacted if you do that. 
So I wonder if we can get more information about the virttest.remote changes. Is there an issue or something to track the changes?

Reply:
@pevogam  Would you please review it again? Thanks a lot!

Reply:
Hi, sorry I somehow missed your last comment.

> No, I think RemoteRunner.run() is the only way to manage services with session via create_service() for now.

If you mean system services than we are just adding a [utility](https://github.com/avocado-framework/aexpect/pull/54/files#diff-a5831b347e80eb0db7ab3201c357ec5244c2d053d3641457aefe6db58c219147R515) doing this as well (I guess for an unrelated reason).

> BTW, may I ask why you want to deprecate RemoteRunner? We have 11 files in tp-libvirt which call that, some cases will be impacted if you do that.

If we go anywhere near the plan we will make sure to make everyone know and deprecate it slowly so don't expect any breakage any time soon. That doesn't mean however that there aren't better options to move to even right now.

> So I wonder if we can get more information about the virttest.remote changes. Is there an issue or something to track the changes?

The basic PR where the `remote` module was moved to aexpect (where it belongs together with all session handling) was the one where we first wondered how many people are using the RemoteRunner and decided to keep it as a leftover utility in VT that we might one day deprecate.

In your particular case I still don't see anything special about using the RemoteRunner other than the additional `CmdResult` which could either be added as a shared wrapper of a session result or the API of a few places could simply be adapted to use a command status and output as two separate variables.

Reply:
@pevogam Thanks for you reply! I understand your concerns, but I'm wondering if we can merge this pr if the schedule of deprecating RemoteRunner is not fixed.  It has been created for one month and a high priority case's failed by it.

@dzhengfy I know you have a lot of experience in remote session operations, could you please share your thought?